src/routes/Dashboard/index.tsx

```diff
@@ -98,29 +105,40 @@ const Dashboard = ({ match }: Props) => {
                   archived={archived.total}
                   width='32%'
                 />
+
                 <ActualityWidget
                   title='Libraries Actuality'
                   width='32%'
                   outdated={major.length}
                   total={libraries.length}
                 />
+
                 <RecentUpdates libraries={recentlyUpdated} width='32%' />
               </WidgetContainer>
             )
 
-            const renderLibraries = () => (
-              <NodeLibrariesTable
-                libraries={uniqueLibraries}
-                outdates={outdates}
-              />
-            )
+            const renderLibraries = () => {
+              const filtered = uniqueLibraries.filter(
+                ({ package: { name } }: any) => name.indexOf(search) > -1
```

Comment:
I believe that `name.includes(search)` is more readable.

Reply:
Totally... never know if I can use it, if polyfills are in place... will do ;)